juddges/data/pl_court_api.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any
+
+import requests
+import xmltodict
+
+
+class PolishCourtAPI:
+    def __init__(self) -> None:
+        self.url = "https://apiorzeczenia.wroclaw.sa.gov.pl/ncourt-api"
+
+    def get_number_of_judgements(self, params: dict[str, Any] | None = None) -> int:
+        if params is None:
+            params = {}
+
+        params |= {"limit": 0}
```

Comment:
Do you want to overwrite "limit" ?

Reply:
Yes, I need to get number of judgements but without fetching data of any judgement, so `limit` must be 0

Reply:
But someone can pass the limit param, and you don't inform/warn them about overwriting.

Reply:
Imo, just add assert.

Reply:
No, that's fully correct. By construction this limit must be 0, and passing won't affect results

Reply:
I'll add warning